enctests/test_configs/dnxhd_color_tests.yml

```diff
@@ -57,9 +58,109 @@ test_dnxhd_color:
       #-colorspace: rgb
       #-pix_fmt: gbrp10le
```

Comment:
I think we should remove commented lines like this and rather rely on git diffs to see what we changed